sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -238,6 +242,203 @@ describe(describeProcessorRootTest(QueueNames.SFASIntegration), () => {
     },
   );
 
+  it(
+    "Should not add SFAS individual data records and display errors when importing invalid data from SFAS " +
```

Comment:
Again we can be specific here about what is invalid.

Reply:
https://github.com/bcgov/SIMS/pull/3633#discussion_r1720058678
https://github.com/bcgov/SIMS/pull/3633#discussion_r1720046437
Having both comments considered, if we want to be specific about what is invalid, then we should keep the logging implementation in this PR and we need to separate the 3 errors into 3 text files. Please confirm that's necessary and intended.